src/plugins/discourse/weights.test.js

```diff
@@ -32,6 +60,46 @@ describe("plugins/discourse/weights", () => {
     });
   });
 
+  describe("_categoryWeight", () => {
+    it("returns the default weight when the category doesn't have a configured weight", () => {
+      expect(_categoryWeight(1, getWeights(getConfig()))).toBe(1);
+    });
+    it("returns the default weight when the category doesn't have a configured weight", () => {
+      const config = getConfig({
+        defaultCategoryWeight: 1,
+        categoryWeights: {"1": 5},
+      });
+      expect(_categoryWeight(1, getWeights(config))).toBe(5);
+    });
+    it("can return a weight set to 0", () => {
+      const config = getConfig({
+        defaultCategoryWeight: 1,
+        categoryWeights: {"1": 0},
+      });
+      expect(_categoryWeight(1, getWeights(config))).toBe(0);
+    });
+  });
+
+  describe("_tagWeight", () => {
+    it("returns the default weight when the category doesn't have a configured weight", () => {
+      expect(_weightFromTags(["tag"], getWeights(getConfig()))).toBe(1);
+    });
+    it("returns the default weight when the category doesn't have a configured weight", () => {
+      const config = getConfig({
+        defaultTagWeight: 1,
+        tagWeights: {"you're": 5},
+      });
+      expect(_weightFromTags(["you're"], getWeights(config))).toBe(5);
```

Comment:
Please set the default weight to something other than 1 so this test can catch if we just are returning hardwired 1